docs/core/plugins/sieve_extprograms.md

```diff
@@ -50,10 +52,8 @@ and the directly executed binaries. The socket directory is searched
 first. Since the use of "/" in program names is prohibited, it is not
 possible to build a hierarchical structure.
 
-Directly forked programs are executed with a limited set of environment
-variables: `HOME`, `USER`, `SENDER`, `RECIPIENT`, and `ORIG_RECIPIENT`.
-Programs executed through the script-pipe socket service currently have
-no environment set at all.
+Programs are executed with a limited set of environment variables: `HOME`,
+`USER`, `SENDER`, `RECIPIENT`, and `ORIG_RECIPIENT`.
```

Comment:
And import_environment? or not?

Reply:
I checked: import_environment is not used.

Reply:
Could add such sentence here to make it clear.

Reply:
added